### ISSUE TITLE
Krennic, Death Star Commandant issues

### ISSUE THREAD

**Issue by Decim8or** [PlayersCommittee/gemp-swccg-public#374]
- [x] Did you search through the list of **open issues** _first_ to ensure that there is not already an report for this issue?

## **To Reproduce**
3. 
4.
5. See error

**Comment by Decim8or** [comment]
I hope that I am posting these bugs I came across in the correct place. New to this forum and the GEMP game system (but I am ADDICTED <3) and want to help make this beautiful project better.

During my most recent game I found the following:

Krennic, Death Star Commandant cannot be targeted to be downloaded by Operational as Planned (V) even though he is a Krennic.

Grand Moff Tarkin (V) does not recognize Krennic, Death Star Commandant as a "Leader" to activate his game text (I mean, neat that is true to the films :lol: but in game kind of important) to add a battle destiny when with an Imperial leader.

I thought there was an error with Corporal Grenwick and his Death Star Sentries, but then I forgot about the CUMULATIVE rule. It's been a while ;)

That's all I can remember that went buggy. Hope this helps!

**Comment by Decim8or** [comment]
I hope that I am posting these bugs I came across in the correct place. New to this forum and the GEMP game system (but I am ADDICTED <3) and want to help make this beautiful project better.

During my most recent game I found the following:

Krennic, Death Star Commandant cannot be targeted to be downloaded by Operational as Planned (V) even though he is a Krennic.

Grand Moff Tarkin (V) does not recognize Krennic, Death Star Commandant as a "Leader" to activate his game text (I mean, neat that is true to the films :lol: but in game kind of important) to add a battle destiny when with an Imperial leader.

I thought there was an error with Corporal Grenwick and his Death Star Sentries, but then I forgot about the CUMULATIVE rule. It's been a while ;)

That's all I can remember that went buggy. Hope this helps!
